tests/test_kubearmor.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pytest
+import platform
+
+
+from utils import (
+    kubectl,
+    microk8s_enable,
+    microk8s_disable,
+    microk8s_reset,
+    wait_for_installation,
+    wait_for_pod_state,
+)
+
+
+class TestKubearmor(object):
+    @pytest.mark.skipif(platform.machine() == "s390x", reason="Not available on s390x")
+    def test_kubearmor(self):
+        """
+        Sets up and validates kubearmor.
+        """
+        print("Enabling Kubearmor")
+        microk8s_enable("kubearmor")
+        print("Validating Kubearmor")
+        self.validate_kubearmor()
+        print("Disabling Kubearmor")
+        microk8s_disable("kubearmor")
+        microk8s_reset()
+
+    def validate_kubearmor(self):
+        """
+        Validate kubearmor by applying policy to nginx container.
+        """
+
+        wait_for_installation()
+        kubearmor_pods = [
+            "kubearmor-controller",
+            "kubearmor",
+            "kubearmor-relay",
+        ]
+        for pod in kubearmor_pods:
+            wait_for_pod_state(
+                "", "kube-system", "running", label="kubearmor-app={}".format(pod)
+            )
+
+        here = os.path.dirname(os.path.abspath(__file__))
+        manifest = os.path.join(here, "templates", "kubearmor-nginx.yaml")
+        policy = os.path.join(here, "templates", "kubearmor-policy.yaml")
+        kubectl("apply -f {}".format(manifest))
+        wait_for_pod_state("", "kubearmor-test", "running", label="app=nginx-test-pod")
+        kubectl("apply -f {}".format(policy))
+        output = kubectl("exec -n kubearmor-test nginx -- apt")
```

Comment:
The test is failing because of this line. Here we go into the nginx pod and call `apt`. Apt returns a non-zero code because it was not asked to do anything. The script the retries the command and eventually times out:
```
Retrying /snap/bin/microk8s.kubectl exec -n kubearmor-test nginx -- apt
```

What is this apt command for? Is it needed?